app/api/v2/handlers/operation_api.py

```diff
@@ -174,19 +193,43 @@ async def update_operation_link(self, request: web.Request):
         link = await self._api_manager.update_operation_link(operation_id, link_id, data, access)
         return web.json_response(link)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Creates a potential Link',
+                          description='Creates a potential link to be executed by an agent. Create a potential Link using '
+                                      'the format provided in the `LinkSchema`. The request body requires `paw`, '
+                                      '`executor`, and `ability`.',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'UUID of the operation object for the link to be created on.'
+                          }])
     @aiohttp_apispec.request_schema(LinkSchema)
-    @aiohttp_apispec.response_schema(LinkSchema)
+    @aiohttp_apispec.response_schema(LinkSchema,
+                                     description='Response contains a completed Link object.')
     async def create_potential_link(self, request: web.Request):
         operation_id = request.match_info.get('id')
         access = await self.get_request_permissions(request)
         data = await request.json()
         potential_link = await self._api_manager.create_potential_link(operation_id, data, access)
         return web.json_response(potential_link)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Retrieve links for an operation.',
```

Comment:
This might be a little confusing because we also have a GET /links endpoint - can you change this to "Retrieve potential links for an operation"?